blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -58,30 +72,89 @@ export const GalleryPresentation = ({
-			resizerURL={resizerURL}
-			ansId={id}
-			ansHeadline={headlines?.basic ? headlines.basic : ""}
-			expandPhrase={phrases.t("global.gallery-expand-button")}
+		<Carousel
+			id={id}
+			className={BLOCK_CLASS_NAME}
+			showLabel
+			label={headlines?.basic ? headlines.basic : ""}
+			slidesToShow={1}
+			showAdditionalSlideControls
+			pageCountPhrase={
+				/* istanbul ignore next */ (current, total) =>
+					phrases.t("global.gallery-page-count-text", { current, total })
+			}
+			enableAutoplay
+			startAutoplayIcon={<Icon name="Play" />}
+			startAutoplayText={phrases.t("global.gallery-autoplay-button")}
+			stopAutoplayIcon={<Icon name="Pause" />}
+			stopAutoplayText={phrases.t("global.gallery-pause-autoplay-button")}
 			autoplayPhraseLabels={{
 				start: phrases.t("global.gallery-autoplay-label-start"),
 				stop: phrases.t("global.gallery-autoplay-label-stop"),
 			}}
-			autoplayPhrase={phrases.t("global.gallery-autoplay-button")}
-			pausePhrase={phrases.t("global.gallery-pause-autoplay-button")}
-			controlsFont={getThemeStyle(arcSite)["primary-font-family"]}
-			pageCountPhrase={
-				/* istanbul ignore next */ (current, total) =>
-					phrases.t("global.gallery-page-count-text", { current, total })
+			enableFullScreen
+			fullScreenShowButton={
+				<button type="button">
+					<Icon name="Fullscreen" />
+					{phrases.t("global.gallery-expand-button")}
+				</button>
+			}
+			fullScreenMinimizeButton={
+				<button type="button">
+					<Icon name="Close" />
+				</button>
+			}
+			adElement={/* istanbul ignore next */ <AdBlock />}
+			adInterstitialClicks={interstitialClicks}
+			nextButton={
+				<Carousel.Button id={id} label="Next Slide">
+					<Icon className={`${BLOCK_CLASS_NAME}__track-icon`} fill="white" name="ChevronRight" />
+				</Carousel.Button>
+			}
+			previousButton={
+				<Carousel.Button id={id} label="Previous Slide">
+					<Icon className={`${BLOCK_CLASS_NAME}__track-icon`} name="ChevronLeft" />
+				</Carousel.Button>
 			}
-			adElement={/* istanbul ignore next */ () => <AdBlock />}
-			interstitialClicks={interstitialClicks}
-			displayTitle={!hideTitle}
-			displayCaption={!hideCaption}
-			displayCredits={!hideCredits}
-		/>
+		>
+			{contentElements.map((galleryItem, itemIndex) => (
+				<Carousel.Item
+					label={phrases.t("global.gallery-page-count-text", { itemIndex, galleryLength })}
+					key={`gallery-item-${galleryItem.url}`}
+				>
+					<MediaItem
+						caption={!hideCaption ? galleryItem.caption : null}
+						credit={
+							!hideCredits ? formatCredits(galleryItem.vanityCredits || galleryItem.credits) : null
+						}
+						title={!hideTitle ? galleryItem.subtitle : null}
+					>
+						<div className={`${BLOCK_CLASS_NAME}__image-wrapper`}>
+							<Image
+								src={imageANSToImageSrc(galleryItem)}
+								resizerURL={resizerURL}
+								resizedOptions={{ auth: galleryItem.auth[resizerAppVersion] }}
+								// 16:9 aspect ratio
+								width={375}
```

Comment:
Seems a little small for the sizes we are outputting - we are saying the image is always `100vw` - What do you think of using the 500 size here?

Reply:
500 width here?

Reply:
updated the width to be wider. also reduced 100vh based off of 75vh max